### PR TITLE
nextjs web extraction example

### PR DESCRIPTION
**Pull request by jvdownie**

### CODE REVIEW

File: nextjs/chat/README.md
Comment (kirk-marple) [inline nextjs/chat/README.md:1]: This references 'graphlit-chat' which isn't correct.

File: nextjs/web-extraction/README.md
Comment (kirk-marple) [inline nextjs/web-extraction/README.md:1]: Same here in the README, the hyperlink goes to 'graphlit-chat' project name.

**Review by kirk-marple — CHANGES_REQUESTED** [review]

File: nextjs/chat/README.md
Comment (jvdownie) [reply nextjs/chat/README.md:1]: this is the chat readme. I did a line to match web-extraction

File: nextjs/web-extraction/README.md
Comment (jvdownie) [reply nextjs/web-extraction/README.md:1]: updated

File: nextjs/chat/README.md
Comment (kirk-marple) [reply nextjs/chat/README.md:1]: Oh oops, didn't notice that was in different folder.

**Review by kirk-marple — APPROVED** [review]

**Pull request merged**